release/yum-repo/earthly.spec

```
@@ -41,7 +41,16 @@ then
     echo "docker was not found; skipping earthly bootstrap"
     exit
 fi
+
 if ! docker info 2>/dev/null >/dev/null
```

I think we probably want to check for `podman` too - `earthly bootstrap` is probably _more_ important with `podman`, since it communicates over `tls` and will need to generate certificates.
```suggestion
if ! ( docker info || podman info ) 2>/dev/null >/dev/null
```

---

Similarly, we probably want to include podman in the `if ! command -v ...` on line 39

---

good point! I'll adjust that in all our scripts.